src/open_inwoner/templates/pages/profile/me.html

```diff
@@ -19,7 +19,6 @@ <h1 class="h1 tabled__title">{% trans "Welkom" %}, {{ user.first_name }}</h1>
     <div class="tabled__section">
         <h2 class="h2 title" id="personal-overview">{% trans "Persoonlijke gegevens" %}
             <div class="tabled__item--force-right tabled__item--mobile-big">
-                {% button href="profile:edit" text=_("Bewerken") icon="edit" transparent=True %}
```

Comment:
Shouldn't this remain as-is?

Reply:
Indeed. Fixed by resolving merge conflict.